### PR TITLE
create classes for api and add methods in ConfigFileReader

### PR DESCRIPTION
**Pull request by Cxristian**
create class restClientApi and add methods in ConfigFileReader

### CODE REVIEW

File: src/main/java/core/report/GeneratorReport.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        final Configuration configuration = new Configuration(reportOutputDirectory, projectName);
+        // additional metadata presented on main page
+        configuration.addClassifications("Platform", "WINDOWS");
+        configuration.addClassifications("Browser", WebDriverConfig.getInstance().getBrowser().toString());
+        configuration.addClassifications("Branch", "RELEASE/1.0");
+        final ReportBuilder reportBuilder = new ReportBuilder(jsonFiles, configuration);
+        final Reportable result = reportBuilder.generateReports();
+        // and here validate 'result' to decide what to do if report has failed
+    }
+}
```
Comment (titor-oopart) [inline src/main/java/core/report/GeneratorReport.java]: add line

File: src/main/java/pivotaltracker/api/restclient/Authentication.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    /**
+     * Constructor.
+     */
+    private Authentication() {
+        accessToken = ConfigFileReader.getInstance().getAccessToken();
+        apiUrl = ConfigFileReader.getInstance().getUrlApi();
+        contentType = ConfigFileReader.getInstance().getContentType();
+    }
+}
```
Comment (titor-oopart) [inline src/main/java/pivotaltracker/api/restclient/Authentication.java]: add line

File: src/test/java/RunCukesTest/RunCukesTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    /**
+     * The method afterExecution executes the project and generate it report.
+     */
+    @AfterTest
+    public void afterExecution() {
+        WebDriverManager.getInstance().getWebDriver().close();
+        GeneratorReport.getInstance().generateReport();
+    }
+}
```
Comment (titor-oopart) [inline src/test/java/RunCukesTest/RunCukesTest.java]: add line

File: pivotaltracker.properties [hunk 4]
```diff
@@ -2,11 +2,10 @@
...
 waitSleepTime=500
-browser=chrome
 
 baseUrl=https://www.pivotaltracker.com/
 
 #api
-apiUrl=services/v5/projects
+apiUrl=services/v5/
 accessToken=2e399ae84895eef55d322156e42f5c41
 contentType=application/json
```
Comment (titor-oopart) [inline pivotaltracker.properties]: add line

File: src/main/java/pivotaltracker/api/restclient/Authentication.java (hunk 2 above)
Comment (LimbertVargas) [inline src/main/java/pivotaltracker/api/restclient/Authentication.java]: add blank space please

File: src/test/java/RunCukesTest/RunCukesTest.java (hunk 3 above)
Comment (LimbertVargas) [inline src/test/java/RunCukesTest/RunCukesTest.java]: add an additional blank space please

File: src/main/java/core/report/GeneratorReport.java (hunk 1 above)
Comment (LimbertVargas) [inline src/main/java/core/report/GeneratorReport.java]: add additional blank space please

**Review by LimbertVargas — COMMENTED** [review]
read the comments please

**Review by titor-oopart — APPROVED** [review]

**Pull request merged**